src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.Custom.pkgdef

```diff
@@ -59,4 +59,4 @@
 "Description"="Use precise semantic token ranges when requesting semantic token information on razor files. Note: This is experimental, some actions may not apply correctly, or at all!"
 "Value"=dword:00000000
 "Title"="Use precise semantic token ranges when requesting semantic token information on Razor files (requires restart)"
-"PreviewPaneChannels"="*"
+"VisibleToInternalUsersOnlyChannels"="*"
```

Comment:
nit: is there any reason to not allow this for external users?

Reply:
it's an experimental feature and we want to enable it for internals until we can confirm it results in better performance.

Reply:
That's fine, but my thinking is that if we're doing targeted notification to enable eventually for public as well (once we verify internal) then we'd still want a user to be able to turn this off if they are experiencing something problematic. If it's only visible to internal than is one of the following true?

1. We can turn this on for external users
2. External users will not see the button to enable/disable

If both are true it seems likely we'd just want to leave it public. Otherwise you'd have to make a code change to enable this to public users instead of just a TN change.